src/transeq.f90

```diff
@@ -836,10 +834,8 @@ subroutine scalar_transport_eq(dphi1, rho1, ux1, uy1, uz1, phi1, schmidt, is_eve
        tb2(:,:,:) = uy2(:,:,:) * tc2(:,:,:)
     endif
     if (skewsc) then
-      ! (1-0.5) * uy.dTdy + 0.5 * (d(uy.T)dy - T.duydy)
+      ! (1-0.5) * uy.dTdy + 0.5 * d(uy.T)dy
```

Comment:
should be (one-half) and half

Reply:
This is fixed in the updated version of this PR